iroha_data_model/Cargo.toml

```diff
@@ -27,6 +27,7 @@ iroha_crypto = { path = "../iroha_crypto" }
 iroha_derive = { version = "=0.1.0", path = "../iroha_macro/iroha_derive" }
 iroha_macro = { version = "=0.1.0", path = "../iroha_macro" }
 iroha_version = { version = "=0.1.0", path = "../iroha_version" }
+iroha_actor = { version = "=0.1.0", path = "../iroha_actor" }
```

Comment:
Do we need it in data model?

Reply:
Yes, for some `declare_versioned!()` parts.

Reply:
Things in data model dont have to be messages. They can be wrapped in type wrappers which can be messages